optimizely/optimizely_user_context.py

```diff
@@ -114,3 +147,132 @@ def as_json(self):
             'user_id': self.user_id,
             'attributes': self.get_user_attributes(),
         }
+
+    def set_forced_decision(self, decision_context, decision):
+        """
+        Sets the forced decision for a given decision context.
+
+        Args:
+            decision_context: a decision context.
+            decision: a forced decision.
+
+        Returns:
+            True if the forced decision has been set successfully.
+        """
+        if not self.client.config_manager.get_config():
```

Comment:
can you add isValid if we don't have it in client? and check from there.

Reply:
Done.